src/main/java/src/chapter5/question3/Question3.java

```diff
@@ -0,0 +1,15 @@
+package src.chapter5.question3;
+
+public class Question3 {
+    public static String replaceWithX(StringBuilder string) {
```

Comment:
Maybe change the parameter name since it isn`t a string.

Reply:
The parameter name has been modified